Source/WebCore/Modules/webxr/WebXRFrame.cpp

```diff
@@ -388,14 +389,37 @@ ExceptionOr<bool> WebXRFrame::fillPoses(const Document& document, const Vector<R
 #endif
 
 #if ENABLE(WEBXR_HIT_TEST)
-Vector<RefPtr<WebXRHitTestResult>> WebXRFrame::getHitTestResults(const WebXRHitTestSource&)
+// https://immersive-web.github.io/hit-test/#dom-xrframe-gethittestresults
+ExceptionOr<Vector<Ref<WebXRHitTestResult>>> WebXRFrame::getHitTestResults(const WebXRHitTestSource& source)
 {
-    return { };
+    // 1. If frame’s active boolean is false, throw an InvalidStateError and abort these steps.
```

Comment:
I did a similar thing when implementing WebXR, i.e., copying the spec steps as comments. But I am no longer sure that's a good idea. Specs change and these comments become obsolete as any other comment. I think we should stick to WebKit's coding style and add them only if it's really a must to understand the code.